### PR TITLE
chore(npm): update release command

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

no

### Motivation / Use-Case

to prevent mistakes

### Breaking Changes

no

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "build:client:sockjs": "webpack ./client-src/sockjs/index.js -o client/sockjs.bundle.js --color --config client-src/sockjs/webpack.config.js",
     "build:client": "rimraf ./client/* && npm-run-all -s -l -p \"build:client:**\"",
     "webpack-dev-server": "cd $INIT_CWD && node ../../../bin/webpack-dev-server.js",
-    "release": "standard-version"
+    "release": "git fetch --tags && standard-version"
```
Comment (evilebottnawi) [inline package.json]: Bad idea, better avoid using `git` in scripts.

Maybe we can implement script to ensure all tags up to date?

**Comment by evilebottnawi** [comment]
let's avoid this, maybe we should open issue in `standard-release` about this or write own script using https://github.com/steveukx/git-js, i think we can close issue here

**Comment by hiroppy** [comment]
Yeah, but I have a holiday today so I'll fix this near day.

**Comment by hiroppy** [comment]
Close once

**Pull request closed without merging**